test/npmignore.js

```diff
@@ -0,0 +1,69 @@
+import path from 'path';
+import test from 'ava';
+import mockery from 'mockery';
+import sinon from 'sinon';
+
+let testedModule;
+
+const gitUtilApi = {
+	newFilesSinceLastRelease: async () => {}
+};
+
+const pkgDirApi = {
+	sync: () => {}
+};
+
+const stubPkgDir = sinon.stub(pkgDirApi, 'sync');
+
+test.before(() => {
+	const stubGitUtil = sinon.stub(gitUtilApi, 'newFilesSinceLastRelease');
+
+	mockery.registerAllowable('../source/util');
+	mockery.registerAllowable('../source/npm/util');
+	mockery.registerMock('./git-util', gitUtilApi);
+	mockery.registerMock('pkg-dir', pkgDirApi);
+
+	stubGitUtil.returns(['source/ignore.txt', 'source/pay_attention.txt', '.hg', 'test/file.txt']);
+
+	mockery.enable({
+		useCleanCache: true,
+		warnOnReplace: false,
+		warnOnUnregistered: false
+	});
+
+	testedModule = require('../source/util');
```

Comment:
Can't this be merged into [line 6](https://github.com/sindresorhus/np/pull/456/files/76e6a8021143b44d8fc40dbb1aed4c33108ec1d6#diff-7b0af0698921a2034cefb84ebcae9568R6)? This way, `testedModule` will be a constant, and we won't override it on each test.

Reply:
No, `mockery` has to setup, before the module is loaded.